src/databricks/labs/remorph/reconcile/utils.py

```diff
@@ -0,0 +1,73 @@
+from pyspark.sql import DataFrame
+from pyspark.sql.functions import col, expr, lit
+
+from databricks.labs.remorph.reconcile.recon_config import ReconcileOutput
```

Comment:
move this entire code to `compare.py` I wouldn't name these as utils.

Reply:
moved.